src/lucky/errors.cr

```diff
@@ -13,4 +13,47 @@ module Lucky
       super "Could not find route matching #{@context.request.method} #{@context.request.path}"
     end
   end
+
+  class UnknownAcceptHeaderError < Error
+    getter request
+
+    def initialize(@request : HTTP::Request)
+      accept_header = request.headers["accept"]?
+      super <<-TEXT
+      Lucky couldn't figure out what format the client accepts.
+
+          The client's Accept header: '#{accept_header}'
+
+      You can teach Lucky how to handle this header:
+
+          # Add this in config/mime_types.cr
+          Lucky::MimeType.register "#{accept_header}", :custom_format
```

Comment:
Should this also mention adding to the `accepted_formats`? Or is that not required?

Reply:
oh, wait, I think I see. The below error would mention that for you.

Reply:
Yeah exactly if they don't add it they'll get the error below 👍